### DISCUSSION TITLE
Add logs in /server/health endpoint

### DISCUSSION THREAD

**Discussion by dstoyanoff** [directus/directus#3982] [Feature Requests]
When there is an issue in the health check, the service will just write the following log:
GET /server/health errored with 503

In order to be able to see details on why the health check fails, you have to run it with an admin token, so that the extended result would be displayed.

In Kubernetes environment this is a bit harder to do, since you don't have a direct access to the container, running Directus (of course, there are ways, but require a lot of steps).

I suggest that in case of error in the HealthCheck, Directus will write the admin result to the log, so it's easier to investigate issues. 

If approved, I am willing to try contributing.

**Comment by rijkvanzanten** [comment]
Good idea! Feel free to go ahead with a PR for that.

The health check function itself is here:

https://github.com/directus/directus/blob/main/api/src/services/server.ts#L72

So we'll probably want to import `logger` from `'../logger'` in there, and then use `logger.warn` to write messages to stdout when the health check has a warn/error

**Reply by dstoyanoff** [reply]
Thanks for the guidance, I will submit a PR soon.

**Comment by rijkvanzanten** [comment]
This was added in https://github.com/directus/directus/pull/3993
